tests/integration/modules/test_freezer.py

```diff
@@ -0,0 +1,13 @@
+import pytest
```

Comment:
**issue (blocking):** this file can be removed. it's an artifact from copier when you tell it to create certain module types. in our case, we've already pulled the tests over from the Salt core.